### PR TITLE
fix(frontend): upgrade missing react-hook-form register

### PR DESCRIPTION
**Pull request by lionelB**
fix react-hook-form api changes (controller and register) 
- fix highlight documents  not shown
- error 500 in glossary and information
- theme icons
- theme selection

### CODE REVIEW

File: targets/frontend/src/components/highlights/Form.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ const HighlightsForm = ({
 
     register,
   } = useForm();
+  console.log(content.contentRelations);
```
Comment (m-maillot) [inline targets/frontend/src/components/highlights/Form.js:27]: Un oubli ? :)

**Review by m-maillot — APPROVED** [review]
Look good for me.

I think we can use the useForm hook here (https://react-hook-form.com/get-started#SchemaValidation).

File: targets/frontend/src/components/highlights/Form.js (hunk 1 above)
Comment (lionelB) [inline targets/frontend/src/components/highlights/Form.js:27]:
```suggestion 
```

**Comment by lionelB** [comment]
> I think we can use the useForm hook here (https://react-hook-form.com/get-started#SchemaValidation).
good idea

**Pull request merged**